src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -1843,9 +1813,13 @@ static void setPreviousValueMetadata(ContextElementResponse* notifyCerP)
         mdP = new Metadata(NGSI_MD_PREVIOUSVALUE, previousValueP->type, previousValueP->numberValue);
         break;
 
-      case orion::ValueTypeNone:
+      case orion::ValueTypeNull:
         mdP = new Metadata(NGSI_MD_PREVIOUSVALUE, previousValueP->type, "");
-        mdP->valueType = orion::ValueTypeNone;
+        mdP->valueType = orion::ValueTypeNull;
+        break;
+
+      case orion::ValueTypeNotGiven:
+        LM_E(("Runtime Error (ValueTypeNotGiven value type)"));
```

Comment:
Like before, is it not better with English text here?
*(Especially as `ValueTypeNotGiven` is a bit cryptic)*

Reply:
Fixed in e85892cdb28f2f510fd2b0a38cacbb43082f9613